app/src/main/java/com/xmartlabs/taskloans/Config.kt

```diff
@@ -22,6 +22,10 @@ object Config {
   val DB_NAME = BuildConfig.APP_NAME.toFileName()
 
   const val API_BASE_URL = BuildConfig.API_BASE_URL
+
+  const val URL_SIGN_IN = "signin"
+  const val URL_SIGN_UP = "signup"
+  const val URL_TASKS_USERS = "tasks/1/users"
```

Comment:
I would not define the url in the config file, if not you will have a lot of URLs.
I'd define the wash task id